packages/types/src/user/events.ts

```diff
@@ -0,0 +1,4 @@
+export const CREATED = "user.created"
+export const UPDATED = "user.updated"
+export const INVITE_CREATED = "invite.created"
+export const INVITE_UPDATED = "invite.updated"
```

Comment:
todo: move to utils. Also, you can probably follow the current pattern we have created
```ts
const UserEvents = {
  created: 'user.' + CommonEvents.CREATED
}
```